flow-typed/types/ima-config.js

```diff
@@ -14,7 +14,8 @@
  * @param {string} adTagUrl - Specifies the ad tag url that is requested from the ad server.
  * @param {string} [adsResponse] - Specifies a VAST 2.0 document to be used as the ads response instead of making a request via an ad tag url (if `adTagUrl` is not set).
  * @param {boolean} [debug=false] - If set to true, loads IMA SDK in debug mode.
- * @param {number} [disableMediaPreload=false] - Whether to disable media pre loading while ad is playing. If set to `true`, the player will start loading the content media just after the ad break will end (incase of preroll ad). This will also overrides `config.adsRenderingSettings.restoreCustomPlaybackStateOnAdBreakComplete` no matters its value and sets it to `false`.
+ * @param {boolean} [disableMediaPreload=false] - Whether to disable media pre loading while ad is playing. If set to `true`, the player will start loading the content media just after the ad break will end (incase of preroll ad). This will also overrides `config.adsRenderingSettings.restoreCustomPlaybackStateOnAdBreakComplete` no matters its value and sets it to `false`.
+ * @param {boolean} [playAdsWithMSE=false] - Whether to play ads with media source extension, If set to `true` the player will handle it, `false` it won't work .
```

Comment:
change this to the new flag and explain it in context of what it will do in IMA, as the MSE context here is out of scope and so it's irrelevant